utils/hooks/useEthereum.tsx

```diff
@@ -118,7 +128,7 @@ export const EthereumProvider = ({ children }: Props) => {
 
   const connect = async () => {
     const instance = await web3Modal.connect();
-    const provider = new providers.Web3Provider(instance)
+    const provider = new providers.Web3Provider(instance, 'any')
```

Comment:
I saw `any` as a workaround instead of having to `reload()` but didn't know where to put it.